### DISCUSSION TITLE
How to upgrade from Lotus market to Boost V2

### DISCUSSION THREAD

**Discussion by wisarmy** [filecoin-project/boost#1836] [Storage Provider] [answered]
https://boost.filecoin.io/tutorials/upgrade-from-lotus-to-boost
Documentation is not very clear，`lotus -> boost v1 -> boost v2`
Can i use boost release-v2.1.1 when executing Lotus to Boost v1 migration steps?

**Comment by LexLuthr** [comment, marked as answer]
No you cannot. I have linked Lotus to Boost upgrade steps to the same page. Please reopen the page and let me know if there is still any confusion.

**Reply by wisarmy** [reply]
Lotus has been upgraded to v1.24.0, Impact Lotus migration to boost v1?

Boost Version | Lotus Version | Golang Version
-- | -- | --
v1.5.0 | v1.18.0 | 1.18.x
v1.5.1, v1.5.2, v1.5.3 | v1.18.0, v1.19.0 | 1.18.x
v1.6.0, v1.6.1, v1.6.2-rc1 | v1.20.x | 1.18.x
v1.6.3, v1.6.4 | v1.22.x | 1.18.x
v1.6.2-rc2, v1.7.0-rc1 | v1.21.0-rc1, v1.21.0-rc2 | 1.20.x
v1.7.0, v1.7.1, v1.7.2v1.7.3, v1.7.4 | v1.23.x | 1.20.x

https://boost.filecoin.io/v/v1/getting-started#boost-and-lotus-compatibility-matrix

**Reply by LexLuthr** [reply]
The migration should not be impacted but please make sure that you do not make any deals when you are on Boost v1. Turn off all deals in config after initialising Boost v1. Then complete the migration to v2 and then turn on the deals again.
